frontend/pages/project.tsx

```diff
@@ -0,0 +1,21 @@
+import { GetServerSideProps } from 'next';
+
+const Project: React.FC<{ data: string }> = ({ data }) => {
+  return (
+    <div>
+      <h1>Project</h1>
+      <p>{data}</p>
+    </div>
+  );
+};
+
+export default Project;
+
+export const getServerSideProps: GetServerSideProps = async () => {
+  const data = 'Données côté serveur';
```

Comment:
Don't send `Données côté serveur` but `[]`.
Later, you'll receive an array not a string